### DISCUSSION TITLE
Dialogic 2 -> Dialogic.start() freezes the game momentarily the first time its called

### DISCUSSION THREAD

**Discussion by gabrielf82** [dialogic-godot/dialogic#2050] [Q&A] [answered]
Hi!
In my 3D scene, I’ve incorporated some example code that initiates a test timeline when the player enters a specific Area3D and presses a Key_E. However, I’ve noticed that the application momentarily freezes at the exact point when start("some_timeline") is called. Interestingly, this pause only occurs the first time; subsequent calls do not result in any pause. Is there a way to prevent this initial pause? Could preloading (or loading) something be a potential solution? Or am I doing something wrong?

`if player_contact: `
`     if event is InputEventKey and event.keycode == KEY_E and event.pressed:`
`         Dialogic.start('timeline_test')`
`         get_viewport().set_input_as_handled()`

**Comment by CakeVR** [comment]
Hello, take a look at this: https://github.com/dialogic-godot/dialogic/issues/1943

**Reply by gabrielf82** [reply]
It's the exact issue I'm facing. Thank you!
Although it seems to be still unresoveld. 
Should I remove this post?

**Reply by CakeVR** [reply]
There is no need to remove this post!

**Comment by Jowan-Spooner** [comment, marked as answer]
Cake added an entry to the FAQ:
https://dialogic-docs.coppolaemilio.com/faq.html#i-encounter-a-small-lag-or-freeze-when-starting-the-dialogue

Hope this helps people in the future.
